fastlane/lib/fastlane/version.rb

```diff
@@ -1,3 +1,3 @@
 module Fastlane
-  VERSION = '1.69.0'.freeze
+  VERSION = '1.70.0'.freeze
```

Comment:
It adds the supply feature of uploading multiple apks (which wasn't mentioned in the 1.69.0 release notes and was also broken) :confused: